src/virtual-machine.js

```diff
@@ -543,17 +543,57 @@ class VirtualMachine extends EventEmitter {
     /**
      * Get an SVG string from storage.
      * @param {int} costumeIndex - the index of the costume to be got.
-     * @return {string} the costume's SVG string, or null if it's not an SVG costume.
+     * @return {string|HTMLImageElement} the costume's SVG string if it's SVG,
+     *     an HTMLImageElement if it's a PNG, or null if it couldn't be found or decoded.
```

Comment:
Is `HTMLImageElement` the best return type? `ImageData` is another option, for example, but I'm not sure which would be "best" to return from this method.

Reply:
Passed in a data URI